python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -82,8 +107,11 @@ def start_ganga(gangadir_for_test, extra_opts=[]):
         logger.info("Initializing")
         Ganga.Runtime._prog.initEnvironment()
     else:
-        from Ganga.Runtime.Repository_runtime import startUpRegistries
+        # Load the config from the appropriate files
+        load_config_files()
```

Comment:
This is doing the same as the normal Ganga startup. Normally, Ganga calls `GangaProgram.configure()` which, along with a bunch of other things (parsing command line, printing the welcome message, setting logging levels, setting up the spyware config, bootstrapping logging, setting command-line config options, doing some legacy test stuff, doing some `--daemon` stuff) parses the config files and sets their values.

This is pulling out just the "parse the config files and sets their values" part since the rest is rather specific to the `ganga` *application*.